R/reactives.R

```diff
@@ -1583,6 +1708,20 @@ maskReactiveContext <- function(expr) {
 #' @param ignoreNULL Whether the action should be triggered (or value
 #'   calculated, in the case of \code{eventReactive}) when the input is
 #'   \code{NULL}. See Details.
+#' @param skipFirst Whether the action should be run the first time the
```

Comment:
When there's this much information, I'd suggest moving it outside of the documentation for the argument, to the main documentation section.